### ISSUE TITLE
`Index XXX not found` even though the index has been created.

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#3681]
To reproduce, `run_tests.py` with commenting out `time.sleep(...)`.

The ready API returns true before Quickwit is actually ready.

**Comment by fulmicoton** [comment]
The problem is the fact that the ingest queue has not been created right after create index has been called.

It was observed in CI here:
https://github.com/quickwit-oss/quickwit/actions/runs/5651558711/job/15309848879?pr=3677

**Comment by fulmicoton** [comment]
We do not want to fix this issue until the work on the ingest api is done. In the meanwhile, I will add a workaround.

**Comment by rdettai** [comment]
I also stumbled on this issue (https://discord.com/channels/908281611840282624/908281611840282627/1229347129261035530)

**Comment by fulmicoton** [comment]
@rdettai it is supposed to be fixed in `main` :( you are workign on main though I assume?

**Comment by rdettai** [comment]
Yes, I have two CI runs using the `edge` Quickwit image that failed over the weekend with this error:
- https://github.com/quickwit-inc/terraform/actions/runs/8779858958
- https://github.com/quickwit-inc/terraform/actions/runs/8763339955

Unfortunately I still don't have the logs because the CI run tears down everything at the end of the run. I can try to reproduce again with this setup, but it's a very slow process as the error doesn't always occur and the stack takes a lot of time to provision.
